seaworthy/tests-core/test_logs.py

```diff
@@ -249,30 +247,15 @@ class FakeLogsContainer:
     def __init__(self, log_entries, expected_params=None, close_timeout=2):
         self.log_entries = log_entries
         self._seen_logs = []
-        self._expected_stream_params = {
-            'stdout': True,
-            'stderr': True,
-            'follow': True,
-        }
+        self._expected_stream_params = {}
```

Comment:
Is there any point in still having an assertion on the expected stream params?

Reply:
We do still have a test asserting that we pass kwargs through to the docker API. We no longer have our own collection of defaults to make the checking more complicated, though.